fit.py

```diff
@@ -0,0 +1,296 @@
+import os
+import re
+
+import pymongo
+from bson.objectid import ObjectId
+from dotenv import load_dotenv
+from flask import (
+    Flask,
+    flash,
+    redirect,
+    render_template,
+    request,
+    session,
+    url_for,
+)
+from flask_login import LoginManager
+from utilities import paginate
+from validators import validate_recipe
+
+load_dotenv()
+
+app = Flask(__name__)
+# Get secret key
+app.config["SECRET_KEY"] = "qwert"
+login = LoginManager(app)
+
+# Mongo Database for FITNESS POT
+myclient = pymongo.MongoClient(os.environ.get("MONGO_CLUSTER"))
+mydb = myclient["fitness_pot"]
+dish_col = mydb["dish"]
+users_col = mydb["users"]
+collection_names = mydb.list_collection_names()
+
+
+DEBUG_LEVEL = "DEBUG"
+
+# =========
+# HOME PAGE - Display home page featured recipes with
+# image and introductory text only
+# =========
+
+
+@app.route("/", methods=["GET", "POST"])
+def index():
+    print(mydb.dish.find_one({"category": "Choose Category"}))
```

Comment:
Let's have Docstrings as comments for our functions. See the [Docstring Conventions](https://www.python.org/dev/peps/pep-0257/)